.github/workflows/testing.yml

```diff
@@ -6,7 +6,7 @@ on:
   workflow_dispatch:
 
 jobs:
-  linux:
+  run-tests-ce:
```

Comment:
> The feature can be disabled using a build tag [3] 'go_tarantool_ssl_disable'.

I suppose it must be disabled for testing of Community Edition. Otherwise, tests will fail.

Reply:
It's okey. Additional tests with Tarantool EE are run only if a TEST_TNT_EE environment variable exists. Without this, local library integration tests are run, without any Tarantool instance.

Reply:
Okay.

Probably we should describe the option for our users somewhere in documentation.

Reply:
Added, see changes in CONTRIBUTING.md

Reply:
(I'll resolve the discussion: it looks fixed by the note in CONTRIBUTING.md.)